src/utils/aepp-wallet-communication/rpc/wallet-rpc.js

```diff
@@ -69,6 +69,32 @@ const REQUESTS = {
       }
     )
   },
+  [METHODS.bridge] (callInstance, instance, client, { name }) {
+    // Authorization check
+    if (!client.isConnected()) return { error: ERRORS.notAuthorize() }
+
+    // wait for approval
+    client.updateInfo()
+
+    return callInstance(
+      'onNodeBridge',
+      { name },
+      () => {
+        return {
+          result: {
+            url: instance.selectedNode.url,
+            ...(instance.selectedNode.internalUrl &&
+              {
+                internalUrl: instance.selectedNode.internalUrl
```

Comment:
proposing to drop it in favor of https://github.com/aeternity/aepp-sdk-js/issues/1217